index.html

```diff
@@ -5301,17 +5288,47 @@ <h2>
           consent.
         </p>
       </section>
-      <section>
-        <h2>
+      <section class="informative">
+        <h2 id="canmakepayment-protections">
           canMakePayment() protections
         </h2>
         <p data-link-for="PaymentRequest">
-          The <a>canMakePayment()</a> method enables the payee to call
-          <a>show()</a> if the user is ready to take advantage of the API, or
-          to fall back to a legacy checkout experience if not. Because this
-          method shares some information with the payee, user agents are
-          expected to protect the user from abuse of the method, for example,
-          by restricting the number or frequency of calls.
+          The <a>canMakePayment()</a> method enables the payee to determine
+          —before calling <a>show()</a>— whether the user is ready to take
```

Comment:
Should be: ... whether the user agent supports the payment handlers with which the payment request was constructed.